piptools/scripts/compile.py

```diff
@@ -6,39 +6,35 @@
 import sys
 import tempfile
 from pathlib import Path
-from typing import IO, Any, BinaryIO, cast
+from typing import IO, Any, BinaryIO, Literal, cast
 
 import click
 from build import BuildBackendException
-from build.util import project_wheel_metadata
 from click.utils import LazyFile, safecall
 from pip._internal.req import InstallRequirement
 from pip._internal.req.constructors import install_req_from_line
 from pip._internal.utils.misc import redact_auth_from_url
 
 from .._compat import parse_requirements
+from ..build import build_project_metadata
 from ..cache import DependencyCache
 from ..exceptions import NoCandidateFound, PipToolsError
 from ..logging import log
 from ..repositories import LocalRequirementsRepository, PyPIRepository
 from ..repositories.base import BaseRepository
 from ..resolver import BacktrackingResolver, LegacyResolver
-from ..utils import (
-    dedup,
-    drop_extras,
-    is_pinned_requirement,
-    key_from_ireq,
-    parse_requirements_from_wheel_metadata,
-)
+from ..utils import dedup, drop_extras, is_pinned_requirement, key_from_ireq
 from ..writer import OutputWriter
 from . import options
+from .options import ALL_BUILD_DISTRIBUTIONS
```

Comment:
```suggestion
from .options import ALL_BUILD_DISTRIBUTIONS
```

Options imported in `34` line.